fixieai/cli/agent/commands.py

```diff
@@ -497,36 +497,52 @@ def _watch_tunnel():
                         break
                     agent_api.update_agent(func_url=url)
                     console.print(
-                        f"[yellow]Tunnel URL was updated, now serving via {url}[/yellow]"
+                        f"[yellow]Tunnel URL was updated. Now serving via {url}[/yellow]"
                     )
 
             threading.Thread(target=_watch_tunnel, daemon=True).start()
         else:
             agent_api = _ensure_agent_updated(client, agent_id, config)
 
         console.print(
-            f"🦊 Agent [green]{agent_api.agent_id}[/] running locally on {host}:{port}, served via {agent_api.func_url}"
+            f"🦊 Agent [green]{agent_api.agent_id}[/] running locally on {host}:{port}, served via {agent_api.func_url}. To interact with this agent, open another terminal window, and run: `fixie console --agent {agent_api.agent_id}`.`"
         )
 
         # Trigger an agent refresh each time it reloads.
         agent_env[FIXIE_REFRESH_ON_STARTUP] = "true"
-        subprocess.run(
-            [
-                python_exe,
-                "-m",
-                "uvicorn",
-                "fixieai.cli.agent.loader:uvicorn_app_factory",
-                "--host",
-                host,
+        if is_typescript:
+            # Get path from this file to ../../agents-ts/dist/serve-bin.js
+            serve_bin_path = os.path.join(
+                os.path.dirname(__file__), "..", "..", "agents-ts", "dist", "serve-bin.js"
+            )
+            subprocess.run([
+                serve_bin_path,
                 "--port",
                 str(port),
-                "--factory",
-            ]
-            + (["--reload"] if reload else []),
-            env=agent_env,
-            cwd=agent_dir or None,
-        ).check_returncode()
-
+                "--agent",
+                path,
+                "--silentStartup",
+                "--humanReadableLogs",
+                "--refreshMetadataAPIUrl",
+                client.get_refresh_agent_url(config.handle),
```

Comment:
It seems better to let the Python code own this value, rather than the Node re-deriving it.